hot-reloader.js

```diff
@@ -152,6 +152,26 @@ class HotReloader extends Emitter {
     const self = this
     const start = new Date().getTime()
 
+    if (moduleName.endsWith('.html')) {
+      let moduleImportName = System.normalizeSync(moduleName + '!text')
+      let module = System.loads[moduleImportName]
+      let parentModuleName
+      if (module && module.importers && module.importers.length) {
+        parentModuleName = System.loads[moduleImportName].importers[0].name
```

Comment:
@JordanBelford shouldn't we iterate all importers? Won't this cause problems if one html file is imported by more than one file?